src-electron/sdk/matter.js

```diff
@@ -110,17 +106,29 @@ async function computeStorageOptionNewConfig(storagePolicy) {
   }
   return storageOption
 }
-
+/**
+ * This asynchronous function computes and returns the new configuration for a storage policy.
+ *
+ * @param {Object} db - The database instance.
+ * @param {Object} clusterRef - The reference to the cluster.
+ * @param {String} storagePolicy - The current storage policy.
+ * @param {Array} forcedExternal - An array of external options.
+ * @param {String} attributeName - The name of the attribute.
+ *
+ * The function first queries to get the cluster name using the cluster reference.
+ * Then it iterates over each option in the forcedExternal array. If the option's category
+ * matches the cluster name and the option's label matches the attribute name, it updates
+ * the storage policy to attributeAccessInterface. Finally, it returns the updated storage policy.
+ */
 async function computeStoragePolicyNewConfig(
   db,
   clusterRef,
   storagePolicy,
   forcedExternal,
   attributeName
 ) {
-  let clusterName
-  clusterName = await queryCluster.selectClusterName(db, clusterRef)
-  forcedExternal.map((option) => {
+  let clusterName = await queryCluster.selectClusterName(db, clusterRef)
+  forcedExternal.some((option) => {
```

Comment:
I don't get this .some(). Why are we using .some()? 
why are we not doing `for ( ... of ...)` ?

Reply:
If we are using .some(), we should properly return a value back.